Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -927,9 +927,12 @@ static NavigatingToAppBoundDomain schemeOrDomainIsAppBound(const String& host, c
 String WebsiteDataStore::resolvedContainerCachesWebContentDirectory()
 {
     if (m_resolvedContainerCachesWebContentDirectory.isNull()) {
+#if !PLATFORM(VISION)
         if (!isPersistent())
             m_resolvedContainerCachesWebContentDirectory = emptyString();
-        else {
+        else
```

Comment:
I'm confused. Doesn't this mean that ephemeral sessions will leak WebXR caches to disk? Why is that OK?

Reply:
This is a good point. Dismissing my approval for now. Agreed allowing WebXR to write to the cache in ephemeral sessions seems questionable.

Reply:
Good point. The other option was to allow the sandbox to read the cache in the normal location, which also might be bad. I'll ask @brentfulgham